Product/SoapUI_Test/ValidationSuite/ConnectValidation-soapui-project.xml

```diff
@@ -4300,7 +4300,7 @@ context.withSql('PatientCorrelationDB') { sql ->
 </soap:Envelope>]]></con:request><con:assertion type="SOAP Fault Assertion" id="9ef901b6-3d34-4e66-b483-d2b10008e6d6" name="Not SOAP Fault"/><con:assertion type="SOAP Response" id="20dc9b73-2174-4430-a2fb-1ed8b6a144e6"/><con:assertion type="XPath Match" id="29f542bd-6425-4e0d-9429-c5911d723d36" name="XPath Match - count(PatientLocationResponse)"><con:configuration><path>declare namespace ns1='urn:ihe:iti:xcpd:2009'; declare namespace ns3="urn:gov:hhs:fha:nhinc:common:nhinccommonentity";
 
 count(//ns3:RespondingGateway_PatientLocationQueryResponse/ns1:PatientLocationQueryResponse/ns1:PatientLocationResponse)</path><content>2</content><allowWildcards>false</allowWildcards><ignoreNamspaceDifferences>false</ignoreNamspaceDifferences><ignoreComments>false</ignoreComments></con:configuration></con:assertion><con:credentials><con:authType>No Authorization</con:authType></con:credentials><con:jmsConfig JMSDeliveryMode="PERSISTENT"/><con:jmsPropertyConfig/><con:wsaConfig mustUnderstand="NONE" version="200508"/><con:wsrmConfig version="1.2"/></con:request></con:config></con:testStep><con:testStep type="delay" name="Delay - adding PLQ-response correlation-table" id="67f21d9d-aa6a-45f2-9856-695e3fe68348"><con:settings/><con:config><delay>3000</delay></con:config></con:testStep><con:testStep type="groovy" name="verified rslData in patientCorrelation" id="aa4f769a-8785-42f0-9738-b7c48fe5e552"><con:settings/><con:config><script>context.withSql('PatientCorrelationDB') { sql -> 
-	def sqlQuery = 'select count(*) from (select * from correlatedidentifiers where PatientAssigningAuthorityId="1.1" and PatientId="38273N244" and CorrelatedPatientAssignAuthId="1.1" and CorrelatedPatientId="111224" and RlsId="1.1"  union select * from correlatedidentifiers where PatientAssigningAuthorityId="1.1" and PatientId="38273N244" and CorrelatedPatientAssignAuthId="4.4" and CorrelatedPatientId="111225" and RlsId="4.4") foundRec' 
+	def sqlQuery = 'select count(*) from (select * from correlatedidentifiers where PatientAssigningAuthorityId="2.2" and PatientId="38273N244" and CorrelatedPatientAssignAuthId="1.1" and CorrelatedPatientId="111224" and RlsId="2.2"  union select * from correlatedidentifiers where PatientAssigningAuthorityId="2.2" and PatientId="38273N244" and CorrelatedPatientAssignAuthId="4.4" and CorrelatedPatientId="111225" and RlsId="2.2") foundRec' 
```

Comment:
Optional: you can use RemoteAA and LocalAA in the query as well

Reply:
I'll just go ahead with what I have.